lib/ansible/modules/cloud/azure/azure_rm_loadbalancer_facts.py

```diff
@@ -44,6 +44,7 @@
     resource_group:
         description:
             - The resource group to search for the desired load balancer
+        required: true
```

Comment:
How about I just want to list all LB in this subscription? It can accept None value